src/pages/AppLogin.vue

```diff
@@ -61,6 +69,10 @@ async function fetchAndStoreUserInfo() {
   }
 }
 
+function validateEmail() {
+  hasValidEmail.value = !email.value || isValidEmail(email.value)
```

Comment:
The condition is a bit difficult to understand. Can we simplify to the one below?

```suggestion
  hasValidEmail.value = email.value.length !== 0 && isValidEmail(email.value)
```

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/121/commits/7b00c57f2704f07bd58178b721d4f56fff015592